oci.go

```diff
@@ -302,3 +310,38 @@ func noNeedForOutput(detach bool, tty bool) bool {
 
 	return true
 }
+
+func getCgroupsDirPath() (string, error) {
+	if cgroupsDirPath != "" {
+		return cgroupsDirPath, nil
+	}
+
+	f, err := os.Open("/proc/self/mountinfo")
+	if err != nil {
+		return "", err
+	}
+	defer f.Close()
+
+	var cgroupRootPath string
+	scanner := bufio.NewScanner(f)
+	for scanner.Scan() {
+		text := scanner.Text()
+		fields := strings.Split(text, " ")
```

Comment:
It would be a good idea to check `len(fields)` so you know that `fields[4]` below is valid.